### ISSUE TITLE
Multiple functions only creates one lambda_code.zip

### ISSUE THREAD

**Issue by fnavarrodev** [transcend-io/terraform-aws-lambda-at-edge#5]
Hi I'm using your module to apply in AWS 2 lambda functions: one for origin request and another for viewer request. But I only get one lambda_code.zip so the source code for both functions ends up being the same. Maybe I'm doing something wrong?

```
    lambda_function_association {
      event_type   = "viewer-response"   
      include_body = false
      lambda_arn   = module.viewer_request_front_lambda.arn
    }
    lambda_function_association {
      event_type   = "origin-response"   
      include_body = false
      lambda_arn   = module.origin_request_front_lambda.arn
    }   
```

......

```
module "viewer_request_front_lambda" {
  source                 = "transcend-io/lambda-at-edge/aws"
  version                = "0.2.1"
  name                   = "viewer_request_front"
  description            = "Basic Auth and prerender headers"
  runtime                = "nodejs10.x"
  lambda_code_source_dir = "${path.module}/../../../src/viewer_request_front"
  s3_artifact_bucket     = "build-us-east-1.mydomain.com"
}

module "origin_request_front_lambda" {
  source                 = "transcend-io/lambda-at-edge/aws"
  version                = "0.2.1"
  name                   = "origin_request_front"
  description            = "Geo Location, prerender redirect and other rewrites"
  runtime                = "nodejs10.x"
  lambda_code_source_dir = "${path.module}/../../../src/origin_request_front"
  s3_artifact_bucket     = "build-us-east-1.mydomain.com"
}
```

**Comment by fnavarrodev** [comment]
this change `output_path = "${var.name}.zip"` in main.tf fixes the problem

**Comment by air3ijai** [comment]
@fnavarrodev,
> this change `output_path = "${var.name}.zip"` in main.tf fixes the problem

Can you please provide more details how to use this in the module section?
Now we use something like this:
```
module "lambda-at-edge-response" {
  source  = "transcend-io/lambda-at-edge/aws"
  version = "0.3.1"

  description            = "Description"
  lambda_code_source_dir = "${path.module}/files/lambda-response-${var.environment}"
  name                   = "lambda-response-${var.environment}"
  s3_artifact_bucket     = "bucket-for-lambda-cloudfront"
  runtime                = "nodejs12.x"
}
```

When we set `output_path = "lambda-response-${var.environment}.zip"` in the module, we get the error:
```
│ An argument named "output_path" is not expected here.
```

Or your solution works only with example provided in the repository with aws resources?
Can we still use the simple way with
```
module "lambda-at-edge-response" {
  source  = "transcend-io/lambda-at-edge/aws"
  version = "0.3.1"
```

Thank you!

**Comment by air3ijai** [comment]
Aha, probably it is clear. In our case it was just required to updated module version to at least from 0.2.3 to 0.3.0 and now we see different zip archives.
```
module "lambda-at-edge-response" {
  source  = "transcend-io/lambda-at-edge/aws"
  version = "0.3.1"
```

**Comment by dmattia** [comment]
Correct! https://github.com/transcend-io/terraform-aws-lambda-at-edge/pull/13/files was the fix, and v0.3.0 was the first version released with that change 😄
